content/get-started/index.md

```diff
@@ -0,0 +1,144 @@
+---
+title: Get Started with Webpack
+contributors:
+  - bebraw
+sort: 3
+---
+
+## Getting Started
+
+webpack is a tool to build javascript modules in your application. To start using `webpack` from its [cli](/api/cli) or [api](/api/node), follow the [Installation instructions](/get-started/install-webpack).
+webpack simplifies your workflow by quickly constructing a dependency graph of your application and bundling them in the right order. webpack can be configured to customise optimisations to your code, to split vendor/css/js code for production, have a hot reloaded dev server and many such cool features. Learn more about [why you should use webpack](/get-started/why-webpack).
+
+## Creating a bundle
+
+Create a demo directory to try out webpack. [Install webpack](/get-started/install-webpack).
+
+```bash
+mkdir webpack-demo && cd webpack-demo
+npm init -y
+npm install --save-dev webpack
+webpack --help //Shows a list of valid cli commands
```

Comment:
The [Install webpack](https://github.com/kalcifer/webpack.js.org/blob/34fda94a3b336ad5a3360b4b2413366a175a319b/content/get-started/install-webpack.md) link on line 15 gives user choices to choose between installing globally and locally.

The global section says:

> However, this is not a recommended practice. This locks you down to a specific version of webpack and might fail in projects that use a different version. The next section tells you how to install webpack locally in a project.

However, I agree that installing globally looks nicer and is easier for beginners and ad-hoc projects. And also given that webpack CLI will [use the local version of webpack if present](https://github.com/webpack/webpack/blob/9085e1e1882d3cae7cfc635258f81ca5d2261c0a/bin/webpack.js#L8), should the Installation docs (which moved to another section in this PR) be updated to say it’s OK to install webpack both globally and locally?